Diz.Core/serialization/xml_serializer/RomByteEncoding.cs

```diff
@@ -5,6 +5,7 @@
 using System.IO;
```

Comment:
this was all fun but now we might not need it anymore except for importing older project types in.  we'll see, if performance is OK with the straight up XML we can just stick with that.